pkgs/top-level/all-packages.nix

```diff
@@ -23120,8 +23120,6 @@ in
     libpng = libpng12;
   };
 
-  xara = callPackage ../applications/graphics/xara { };
```

Comment:
maybe we should do:
```
xara = lib.throw "xara has been removed from nixpkgs. Unmaintained since 2006" # added 2020-06-24
```

Reply:
> `xara` is also referenced in the following file:
> 
> https://github.com/nixos/nixpkgs/blob/3a44ede7110b47bd8fa8a4dbe027674b0ac81a12/maintainers/scripts/debian-patches.sh#L3-L5

Thanks! I have updated it to your suggested path.

Reply:
> maybe we should do:
> 
> ```
> xara = throw "xara has been removed from nixpkgs. Unmaintained since 2006" # added 2020-06-24
> ```
> 
> in aliases.nix

Added :+1: .